src/isIsogram.test.js

```diff
@@ -6,4 +6,28 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should return true for the word "playgrounds"`, () => {
+    expect(isIsogram('playgrounds')).toBe(true);
+  });
+
+  it(`should return false for the word "look"`, () => {
+    expect(isIsogram('look')).toBe(false);
+  });
+
+  it(`should return false for the word "Adam" (case-insensitive)`, () => {
+    expect(isIsogram('Adam')).toBe(false);
+  });
+
+  it(`should return false  for the word "Oops" (case-insensitive)`, () => {
+    expect(isIsogram('Oops')).toBe(false);
+  });
+
+  it(`should return false for the word "repetitions"`, () => {
+    expect(isIsogram('repetitions')).toBe(false);
+  });
+
+  it(`should return true for the word "isogram"`, () => {
+    expect(isIsogram('isogram')).toBe(true);
+  });
```

Comment:
Consider adding a test case for an empty string to ensure the function correctly identifies it as an isogram, as specified in the task description.